### ISSUE TITLE
Add Deeppy to Python Package Index

### ISSUE THREAD

**Issue by ryanphjohnson** [andersbll/deeppy#25]
This is a fantastic library! I would love to use it, and I'm sure others would as well. Could this be added to the [Python Package Index](https://pypi.python.org/pypi)? The site makes it look like its simple to add to the list. I wouldn't want to take credit for work I didn't do by adding it to the list myself. Could you submit this to be included?

**Comment by andersbll** [comment]
Thank you for the kind words! I plan on cleaning up the codebase soon and 'releasing' a version 0.1. This might be a good opportunity to put it on PyPI. I don't know what to do about CUDArray, though. This library requires a separate installation of the C-based parts.

**Comment by ryanphjohnson** [comment]
Yes, after I looked at the work a bit more, it looks like CUDArray is a pretty integral piece, which I don't know if PyPI would be ok with. Either way, keep up the great work!
